local-infrastructure/settings/keycloak-local-user.sh

```diff
@@ -0,0 +1,116 @@
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort" --rolename "COMFORT User"
+
+echo "...$USERNAME created"
+
+export FIRST="CSST"
+export LAST="ROLE_3"
+export USERNAME=`echo "$FIRST"_"$LAST" | tr A-Z a-z`
+export ROLENAME="Form Reviewer"
+export EMAIL="$USERNAME@email.com"
+
+echo "creating user $USERNAME..."
+
+USER_ID=`kcadm.sh create users -r $REALM -s username=$USERNAME -s firstName=$FIRST -s lastName=$LAST -s email=$EMAIL -i`
+kcadm.sh update users/$USER_ID -r $REALM -s enabled=true
+kcadm.sh set-password -r $REALM --username $USERNAME --new-password password123
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort-agriseafoodopscreening" --rolename "$ROLENAME"
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort-forestrysectoropscreening" --rolename "$ROLENAME"
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort-minesoperatorscreening" --rolename "$ROLENAME"
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort" --rolename "COMFORT User"
+
+echo "...$USERNAME created"
+
+export FIRST="CSST"
+export LAST="ROLE_4"
+export USERNAME=`echo "$FIRST"_"$LAST" | tr A-Z a-z`
+export ROLENAME="Form Viewer"
+export EMAIL="$USERNAME@email.com"
+
+echo "creating user $USERNAME..."
+
+USER_ID=`kcadm.sh create users -r $REALM -s username=$USERNAME -s firstName=$FIRST -s lastName=$LAST -s email=$EMAIL -i`
+kcadm.sh update users/$USER_ID -r $REALM -s enabled=true
+kcadm.sh set-password -r $REALM --username $USERNAME --new-password password123
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort-agriseafoodopscreening" --rolename "$ROLENAME"
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort-forestrysectoropscreening" --rolename "$ROLENAME"
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort-minesoperatorscreening" --rolename "$ROLENAME"
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort" --rolename "COMFORT User"
+
+echo "...$USERNAME created"
+
+export FIRST="CSST"
+export LAST="ROLE_5"
+export USERNAME=`echo "$FIRST"_"$LAST" | tr A-Z a-z`
+export ROLENAME="Request Access"
+export EMAIL="$USERNAME@email.com"
+
+echo "creating user $USERNAME..."
+
+USER_ID=`kcadm.sh create users -r $REALM -s username=$USERNAME -s firstName=$FIRST -s lastName=$LAST -s email=$EMAIL -i`
+kcadm.sh update users/$USER_ID -r $REALM -s enabled=true
+kcadm.sh set-password -r $REALM --username $USERNAME --new-password password123
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort-agriseafoodopscreening" --rolename "$ROLENAME"
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort-forestrysectoropscreening" --rolename "$ROLENAME"
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort-minesoperatorscreening" --rolename "$ROLENAME"
+kcadm.sh add-roles -r $REALM --uusername $USERNAME --cclientid "comfort" --rolename "COMFORT User"
```

Comment:
Might be worth putting this into a bash for loop, and passing in LAST and ROLENAME exports as variables - if we add more forms, this could turn into a copy-paste error-prone zone.

Reply:
replaced with a function.